source/js/components/share-button-group/share-button-group.jsx

```diff
@@ -93,31 +100,51 @@ export default class ShareButtonGroup extends React.Component {
   }
 
   renderLinkButton() {
-    let tooltip = this.state.linkCopied
-      ? `Copied`
-      : `Copy page URL to clipboard`;
-    let label = this.state.linkCopied ? `Copied` : `Copy`;
+    let label = this.state.linkCopied ? (
+      <Localized id="tooltip-copied">{`Copied`}</Localized>
+    ) : (
+      <Localized id="tooltip-copy">{`Copy`}</Localized>
+    );
+    let labelMini = this.state.linkCopied ? (
+      <Localized id="tooltip-page-copied">{`Page link copied`}</Localized>
+    ) : (
+      <Localized id="tooltip-page-copy">{`Copy page link`}</Localized>
+    );
     label =
       this.props.version === `mini` ? (
-        <span class="sr-only">{label} page link</span>
+        <span class="sr-only">{labelMini}</span>
       ) : (
         label
       );
 
     let classes = classNames(`btn btn-secondary btn-share link-share`, {
       copied: this.state.linkCopied
     });
-
-    return (
-      <a
-        class={classes}
-        href="#"
-        onClick={event => this.handleLinkButtonClick(event)}
-        title={tooltip}
-      >
-        {label}
-      </a>
+    let link = this.state.linkCopied ? (
+      <Localized id="link-copied" attrs={{ title: true }}>
+        <a
+          class={classes}
+          href="#"
+          onClick={event => this.handleLinkButtonClick(event)}
+          title="Copied"
```

Comment:
If this is the only difference, why do we need to fully duplicate this block? Surely we can set this property to the correct locale string without the entire block wrapper.

Reply:
I’m not sure we can do this differently, unless we can store a translation in a variable. @stasm do you know?

Reply:
I guess you could do `<Localized id={this.state.linkCopied ? "link-copied" : "link-copy"}>` and then the same for the `title` attribute (or skip it in JSX as Fluent will add it anyways). It probably comes down to a personal preference.

Reply:
Oh, that’s smarter :) Thanks, updated!